src/org/thoughtcrime/securesms/WelcomeActivity.java

```diff
@@ -256,7 +256,7 @@ private File copyToCacheDir(Uri uri) throws IOException {
         }
     }
 
-    private void startQrAccountCreation(String qrCode)
+    private void startQrAccountOrQrLoginCreation(String qrCode)
```

Comment:
"login creation" is a bit weird. you meant `startQrAccountCreationOrLogin`? 

an alternative may be to  not alter the function name at all, "account creation" is still good enough: it creates and adds a `DcAccounts` object - either by creating a new email account or by logging in to an existent. from the view of the ui, this is a minor detail, the "big" thing is that a new `DcAccount` object is created.